test/failure_handler/src/share/conf/mac.properties

```diff
@@ -61,9 +63,21 @@ native.stack.delimiter=\0
 native.stack.params.repeat=6
 native.stack.args=-c\0DevToolsSecurity --status | grep -q enabled && lldb -o 'attach %p' -o 'thread backtrace all' -o 'detach' -o 'quit'
 
+native.jhsdb.app=bash
+native.jhsdb.jstack.delimiter=\0
+native.jhsdb.jstack.args=-c\0DevToolsSecurity --status | grep -q enabled && jhsdb jstack --mixed --pid %p
```

Comment:
AFAICS `jhsdb` does check "Developer mode" on macos before trying to attach and will just report 'lack of privileges' (as opposed to hanging with a modal window asking for permission), so you can move `jshdb.jstack` to `common.properties`.